UI/MediatR/Handlers/GetTuitionPartnerQueryHandler.cs

```diff
@@ -0,0 +1,273 @@
+using Domain;
```

Comment:
As we talked over before - think this is a good idea, but think we need to make sure we do the rest of the site in a similar way.  Worth adding a new tech debt ticket to the board for this?

Reply:
Yes, I do agree. My thinking was given this is possibly one of the simpler pages why not start from this one?

Reply:
One to cover in the dev team chat and make sure it's as a tech debt ticket on the board then?

Reply:
Added new ticket in Jira - 839